perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -76,8 +89,51 @@
 flags.DEFINE_integer('cassandra_stress_retries', 1000,
                      'Number of retries when error encountered during stress.')
 
+# Use "./cassandra-stress help -pop" to get more details.
+# [dist=DIST(?)]: Seeds are selected from this distribution
+#  EXP(min..max):
+#      An exponential distribution over the range [min..max]
+#  EXTREME(min..max,shape):
+#      An extreme value (Weibull) distribution over the range [min..max]
+#  QEXTREME(min..max,shape,quantas):
+#      An extreme value, split into quantas, within which the chance of
+#      selection is uniform
+#  GAUSSIAN(min..max,stdvrng):
+#      A gaussian/normal distribution, where mean=(min+max)/2, and stdev
+#      is (mean-min)/stdvrng
+#  GAUSSIAN(min..max,mean,stdev):
+#      A gaussian/normal distribution, with explicitly defined mean and stdev
+#  UNIFORM(min..max):
+#      A uniform distribution over the range [min, max]
+#  Preceding the name with ~ will invert the distribution,
+#  e.g. ~EXP(1..10) will yield 10 most, instead of least, often.
+flags.DEFINE_enum('cassandra_stress_pop_distribution', None,
+                  ['EXP', 'EXTREME', 'QEXTREME', 'GAUSSIAN', 'UNIFORM',
+                   '~EXP', '~EXTREME', '~QEXTREME', '~GAUSSIAN', '~UNIFORM'],
+                  'The population distribution cassandra-stress uses. '
+                  'By default, use no distribution.')
+
+flags.DEFINE_integer('cassandra_stress_pop_size', None,
+                     'The range of the distribution across all clients. '
+                     'By default, the size of the population equals to '
+                     'max(num_keys,cassandra_stress_preload_num_keys).')
```

Comment:
num_keys is the number keys to read/insert, where same key can be read/insert repeatedly.
But if the pop_size is larger than cassandra_stress_preload_num_keys, this will allow to read/insert non-existant keys, where read from non-existant keys will cause error from cassandra stress.